testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -709,4 +712,42 @@ public void testRenameTableAcrossSchema()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
         assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
     }
+
+    @Test(dataProvider = "objectNamesProvider")
+    public void testCreateSchema(String name, boolean delimited)
+    {
+        if (!hasBehavior(SUPPORTS_CASE_SENSITIVE_IDENTIFIERS)) {
+            throw new SkipException("Skipping since case-sensitive-identifiers is not supported at all");
```

Comment:
If it is not supported then it will work in different way. Can you validate that `CREATE SCHEMA X` creates schema `x`?